docs/release-notes/3.0.0_Alpha.md

```diff
@@ -0,0 +1,137 @@
+
+PMM is now deployed inside containers within AMI and OVF instances. These containers are managed via Podman and operate in rootless mode, which means they do not require root privileges to run. 
+
+This approach significantly increases security and ensures a consistent and unified experience across different deployment types.
+
+### More stable and quicker upgrades
+
+ PMM 3, we're enhancing the stability and speed of the upgrade process by transitioning from RPM updates to exclusive container updates. This streamlined method resolves upgrade glitches, ensures smoother experiences for Docker, Podman, and OVF/AMI setups, and improves our testing workflows.
+
+At the same time, we're maintaining the UI upgrade option by integrating [Watchtower](https://containrrr.dev/watchtower/), an external upgrading tool. When you click the **Upgrade Now** button on the Home Dashboard, Watchtower seamlessly replaces the PMM Server container and links the new one to the existing volume, preserving all data and settings intact.
+
+With the Beta release, Watchtower commands will be seamlessly integrated into the Easy-Install script, allowing for a one-step setup of PMM alongside Watchtower. Until then, you can manually test the PMM installation by following the instructions for [installing PMM Server with Docker container](../install-pmm/install-pmm-server/index.md).
+
+## Improved UX with Grafana latest release
+
+PMM now integrates Grafana 10.4, which delivers the following important enhancements alongside all the advancements introduced since the previous Grafana 9.2.20 integration.
+
+For the full list of changes included in this update, see [Grafana's 10.4 release blog](https://grafana.com/blog/2024/03/06/grafana-10.4-release-all-the-latest-features/).
+
+### Improved navigation
+
+With the 10.4 update, PMM now includes a revamped header with search, breadcrumbs, and a reorganized menu that groups related tools together, making it easier to navigate PMM's features and find what you need:
+
+![New Dashboard page](../_images/Navigation_Dashboard.png)
+
+### Improved Alerting workflow
+
+Leveraging the new Grafana user interface updates, we've taken the opportunity to refine the workflow for creating alerts from the **Alert Rules** and **Alert Rule Templates** pages.
+
+You'll notice separate, more visible options for creating different types of alert rules, cutting down on unnecessary steps and making it easier to manage various alert rules, templates, and configurations:
+
+![New Alerts rule page](../_images/Grafana_Alerting.png)
+
+### Simplified Administration Settings with Dedicated Menus
+
+We've made administration settings easier to manage by introducing a new **Administration** menu. This menu brings together all Grafana-related configurations and account management options that were previously scattered across the general **Configuration** section. This change helps administrators locate and manage Grafana-specific settings more efficiently.
```

Comment:
Just a note, the "Administration" menu (Merged "Configuratio" and "Server Admin") was introduced by Grafana.